docker/docker-compose.prod.yml

```diff
@@ -0,0 +1,18 @@
+# Compose file for use on production servers.
```

Comment:
Yeah, I see what you mean. I'll add a command into *docker-server-configs* repo and get rid of this file. Can use *docker-compose.consul.yml* for reproducing production-like environment.

I'm using compose as a convenience so that I don't have to type/copy long commands every time I need to start up containers, since all the configuration is defined here. That makes more sense when there's more than one service.